var/spack/repos/builtin/packages/berkeleygw/package.py

```diff
@@ -121,9 +121,12 @@ def build(self, spec, prefix):
         if "+mpi" in spec:
             paraflags.append("-DMPI")
 
+        # We need to copy fflags in case we append to it (#34019):
+        fflags = spec.compiler_flags["fflags"][:]
         if "+openmp" in spec:
             paraflags.append("-DOMP")
             spec.compiler_flags["fflags"].append(self.compiler.openmp_flag)
```

Comment:
you forgot to delete the actual offending line :)

Reply:
Thanks, pushed!